ansible/tasks/security/main.yml

```diff
@@ -22,6 +31,14 @@
                          CPUQuota=50% MemoryLimit=512M BlockIOWeight=10
       fi
 
+- name: "Per-user ressource limits: Execute script with pam_exec"
+  lineinfile:
+    create: no
+    path: /etc/pam.d/common-session-noninteractive
+    insertafter: session	required	  pam_systemd.so
```

Comment:
No idea what you mean.
In anycase, the PAM handling seems very brittle to me, but it's very out-of-scope for this PR (as most/all of it is in ldap-nss)

Reply:
Sorry, GitHub formatting messes it up. I'm not sure how well we should be using hardcoded indents, so I was suggesting matching any number of spaces, in case the indents change for whatever reason.

Reply:
Oh, I see what you mean; yes, we ought to do that.